ec/acc/testdata/deployment_basic_defaults_hw_2.tf

```diff
@@ -9,7 +9,16 @@ resource "ec_deployment" "defaults" {
   version                = data.ec_stack.latest.version
   deployment_template_id = "%s"
 
-  elasticsearch {}
+  elasticsearch {
```

Comment:
Does it make sense to modify test checks instead? It looks like the acceptance test (`TestAccDeployment_basic_defaults_hw`) checks updating configuration once the template is changed.
 
Also shall the acceptance test work as expected without these explicit changes to the sizes? The new template (`aws-hot-warm-v2`) has size 4g for both `hot_content` and `warm` tiers so I guess once the template attribute is set to `aws-hot-warm-v2`, the tier sizes should change accordingly even without these explicit settings in the configuration.